app/models/proposal.rb

```diff
@@ -170,6 +171,10 @@ def notifications
     proposal_notifications
   end
 
+  def users_to_notify
```

Comment:
Maybe increasing proposal model spec to check this new method, just reusing existing spec for `voters` could be a nice test improvement https://github.com/rockandror/consul/blob/followable/spec/models/proposal_spec.rb#L382-L425